.github/workflows/localstack.yml

```
@@ -0,0 +1,53 @@
+name: Deploy Domain Protect into LocalStack
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - "**/*.py"
+      - "**/*.tf"
+      - "**/*.txt"
+      - ".github/workflows/localstack.yml"
+
+jobs:
+  localstack:
+    name: localstack
```

Related to https://github.com/domain-protect/terraform-aws-domain-protect/pull/215/files#r1957201927

```suggestion
    name: Deploy
```